<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ui_mainwindow</class>
 <widget class="QMainWindow" name="ui_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSesame</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/experiment.png</normaloff>:/icons/experiment.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="tab_widget" name="tabwidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_recent_files">
     <property name="title">
      <string>Recent files</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/recent_large.png</normaloff>:/icons/recent_large.png</iconset>
     </property>
     <addaction name="actionDummy"/>
    </widget>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="menu_recent_files"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_run">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_run_in_window"/>
    <addaction name="action_run_quick"/>
    <addaction name="separator"/>
    <addaction name="action_enable_auto_response"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_close_current_tab"/>
    <addaction name="action_close_all_tabs"/>
    <addaction name="action_close_other_tabs"/>
    <addaction name="action_onetabmode"/>
    <addaction name="separator"/>
    <addaction name="action_show_overview"/>
    <addaction name="action_show_pool"/>
    <addaction name="action_show_stdout"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_preferences"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_run"/>
  </widget>
  <widget class="QDockWidget" name="dock_overview">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>124</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Overview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_widget_overview">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="tree_overview" name="itemtree">
       <property name="autoExpandDelay">
        <number>4</number>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_pool">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>File pool</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dock_pool_widget">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_stdout">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Debug window</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="console" name="console">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="button_help_console">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Help</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_clear_console">
             <property name="toolTip">
              <string>Clear debug window</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_reset_console">
             <property name="toolTip">
              <string>Reset workspace</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolbar_main">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_run"/>
   <addaction name="action_run_in_window"/>
   <addaction name="action_run_quick"/>
   <addaction name="action_kill"/>
   <addaction name="separator"/>
   <addaction name="action_close_other_tabs"/>
   <addaction name="action_onetabmode"/>
   <addaction name="separator"/>
   <addaction name="action_show_overview"/>
   <addaction name="action_show_pool"/>
   <addaction name="action_show_stdout"/>
  </widget>
  <widget class="toolbar_items" name="toolbar_items">
   <property name="windowTitle">
    <string>Toolbar items</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/open_large.png</normaloff>:/icons/open_large.png</iconset>
   </property>
   <property name="text">
    <string>Open…</string>
   </property>
   <property name="toolTip">
    <string>Open…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/save_large.png</normaloff>:/icons/save_large.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/saveas_large.png</normaloff>:/icons/saveas_large.png</iconset>
   </property>
   <property name="text">
    <string>Save as…</string>
   </property>
   <property name="toolTip">
    <string>Save as…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/quit_large.png</normaloff>:/icons/quit_large.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>Run fullscreen…</string>
   </property>
   <property name="toolTip">
    <string>Run fullscreen…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_close_all_tabs">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/closeothertabs_large.png</normaloff>:/icons/closeothertabs_large.png</iconset>
   </property>
   <property name="text">
    <string>Close all tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="action_close_other_tabs">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/close_large.png</normaloff>:/icons/close_large.png</iconset>
   </property>
   <property name="text">
    <string>Close other tabs</string>
   </property>
   <property name="toolTip">
    <string>Close other tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_run_in_window">
   <property name="text">
    <string>Run in window…</string>
   </property>
   <property name="toolTip">
    <string>Run in window…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_show_pool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/pool_large.png</normaloff>:/icons/pool_large.png</iconset>
   </property>
   <property name="text">
    <string>Show file pool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_show_stdout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="os-debug">
     <normaloff>:/icons/stdout_large.png</normaloff>:/icons/stdout_large.png</iconset>
   </property>
   <property name="text">
    <string>Show debug window</string>
   </property>
   <property name="toolTip">
    <string>Show debug window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_enable_auto_response">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable auto response</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_preferences">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/options_large.png</normaloff>:/icons/options_large.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionDummy">
   <property name="text">
    <string>Dummy</string>
   </property>
  </action>
  <action name="action_show_overview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show overview area</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+\</string>
   </property>
  </action>
  <action name="action_onetabmode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/onetabmode_large.png</normaloff>:/icons/onetabmode_large.png</iconset>
   </property>
   <property name="text">
    <string>One-tab mode</string>
   </property>
   <property name="toolTip">
    <string>One-tab mode</string>
   </property>
  </action>
  <action name="action_run_quick">
   <property name="text">
    <string>Quick run</string>
   </property>
   <property name="toolTip">
    <string>Quick run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="action_close_current_tab">
   <property name="text">
    <string>Close current tab</string>
   </property>
   <property name="toolTip">
    <string>Close current tab</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="action_kill">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Kill experiment</string>
   </property>
   <property name="toolTip">
    <string>Forcibly kill the experiment</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tree_overview</class>
   <extends>QTreeWidget</extends>
   <header>libqtopensesame.widgets.tree_overview.h</header>
  </customwidget>
  <customwidget>
   <class>toolbar_items</class>
   <extends>QToolBar</extends>
   <header>libqtopensesame.widgets.toolbar_items.h</header>
  </customwidget>
  <customwidget>
   <class>console</class>
   <extends>QPlainTextEdit</extends>
   <header>libqtopensesame.console.h</header>
  </customwidget>
  <customwidget>
   <class>tab_widget</class>
   <extends>QTabWidget</extends>
   <header>libqtopensesame.widgets.tab_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_clear_console</sender>
   <signal>clicked()</signal>
   <receiver>console</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>990</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
